Client/Redux/GlobalState/AppState.swift

```diff
@@ -19,6 +19,7 @@ struct AppState: StateType {
                 case (.themeSettings(let state), .themeSettings): return state as? S
                 case (.tabsPanel(let state), .tabsPanel): return state as? S
                 case (.remoteTabsPanel(let state), .remoteTabsPanel): return state as? S
+                case (.fakespot(let state), .fakespot): return state as? S
```

Comment:
This looks like we're fighting with the type system, always have to do castings. Maybe we can improve them with generics.

Reply:
Yes this process feels a bit tedious if you have any ideas to improve please share